packages/pageheader/README.md

```diff
@@ -2,23 +2,68 @@
 
-A PageHeader doesn't have to have actions
+The title or breadcrumb nav may or may not display an icon associated with the current page.
 
-### Placement
-When there is no overflow menu, actions should be ordered right to left from most used to least used.
+![pageHeader-noIcon](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-noIcon.png)
 
-Example:
-[Secondary Action] [Primary Action]
+![pageHeader](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader.png)
 
-<br />
+### Examples
 
-When there is an overflow menu, the overflow menu button is always on the far right.
+![pageHeader-example](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example.png)
 
-Example:
-[Secondary Action] [Primary Action] [⋮]
+## Page header with actions
+
+Some pages have one or more actions that are relevant to the whole page.
+
+For visibility, these are placed at the top of the page next to the title or breadcrumbs.
+
+![pageHeader-withAction](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-withAction.png)
+
+If there is more than one action, the most important action is closest to the right edge of the screen.
+
+![pageHeader-twoActions](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-twoActions.png)
+
+If there are more than three actions, the two most important actions are placed to the left of an overflow menu.
+
+![pageHeader-withOverflow](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-withOverflow.png)
+
+### Examples
+
+![pageHeader-example-oneAction](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example-oneAction.png)
+
+![pageHeader-example-twoActions](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example-twoActions.png)
+
+![pageHeader-example-withOverflowActions](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example-withOverflowActions.png)
+
+## Page header with tabs
+
+Pages that have multiple sections of content use tabs to organize content. Page header tabs are always placed last (bottom) area of the page header.
+
+![pageHeader-tabbedContent](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-tabbedContent.png)
+
+### Examples
+
+![pageHeader-example-withTabs](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example-withTabs.png)
+
+![pageHeader-example-withTabs+action](https://raw.githubusercontent.com/dcos-labs/ui-kit/master/packages/pageheader/readmeImg/pageHeader-example-withTabs+action.png)
+
+## Page header with subheader
```

Comment:
I thought about this, and I think "page header content" could be confusing because "content" could be misinterpreted as _all_ of the content in the page header (breadcrumb, actions, tabs, etc...).

I went with "page header body" instead.